docs/source/developers/bug_reports.rst

```diff
@@ -18,82 +18,70 @@
 .. _bug-reports:
 
 ********************************
-Report bugs and propose features
+Bug reports and feature requests
 ********************************
 
-Using the software and sharing your experience is a very helpful contribution
-itself. Those who actively develop Arrow need feedback from users on what
-works and what doesn't. Alerting us to unexpected behavior and missing features,
-even if you can't solve the problems yourself, help us understand and prioritize
-work to improve the libraries.
+Arrow relies upon user feedback to identify defects and improvement
+opportunities. All users are encouraged to participate by creating bug reports
+and feature request or commenting on existing issues. Even if you cannot
+contribute solutions to the issues yourself, your feedback helps us understand
+problems and prioritize work to improve the libraries.
 
-We use `JIRA <https://issues.apache.org/jira/projects/ARROW/issues>`_
-to manage our development "todo" list and to maintain changelogs for releases.
-In addition, the project's `Confluence site <https://cwiki.apache.org/confluence/display/ARROW>`_
-has some useful higher-level views of the JIRA issues.
+.. _apache-arrow-jira:
 
-To create a JIRA issue, you'll need to have an account on the ASF JIRA, which
-you can `sign yourself up for <https://issues.apache.org/jira/secure/Signup!default.jspa>`_.
-The JIRA server hosts bugs and issues for multiple Apache projects. The JIRA
-project name for Arrow is "ARROW".
+Apache Arrow Jira
+=================
 
-You don't need any special permissions on JIRA to be able to create issues.
-Once you are more involved in the project and want to do more on JIRA, such as
-assign yourself an issue, you will need "Contributor" permissions on the
-Apache Arrow JIRA. To get this role, ask on the mailing list for a project
-maintainer's help.
+The Arrow project uses `JIRA <https://issues.apache.org/jira/projects/ARROW/issues>`_
```

Comment:
```suggestion
The Arrow project uses `Jira <https://issues.apache.org/jira/projects/ARROW/issues>`_
```

Reply:
I know other docs use "JIRA", but Atlassian uses "Jira" - even to the point that if you type "JIRA" into a Jira issue, the web interface will automatically change it to "Jira". I'm a late convert to using "Jira", and I'll use "JIRA" here if that's important - but it's apparently not the preferred, canonical representation.

Reply:
OK.

Reply:
I somehow misread your suggestion - thanks for pointing out the inconsistent (and wrong) use of "JIRA". Applied suggestion.